docs/en/essentials/passing-props.md

```diff
@@ -27,6 +27,13 @@ const User = {
 const router = new VueRouter({
   routes: [
     { path: '/user/:id', component: User, props: true }
+    
+    // for routes with named views, you have to define the props option for each named view:
+    {
+      path: '/user/:id', 
+      components: { default: User, sidebar: Sidebar },
+      props: {user: true, sidebar: true }
```

Comment:
It could be confusing. `sidebar` is okay, it matches a key on components. Where does `user` come from?